components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.http/src/main/java/org/wso2/carbon/event/output/adapter/http/HTTPEventAdapter.java

```diff
@@ -330,7 +330,15 @@ public void run() {
                     }
                 }
 
-                this.getHttpClient().executeMethod(hostConfiguration, method);
+                int responseCode = this.getHttpClient().executeMethod(hostConfiguration, method);
+                if (responseCode == 200 || responseCode == 201 || responseCode ==200) {
+                    log.info("Successfully connect to the endpoint. Relieved HTTP response code is : " +
+                            responseCode);
+                }
+                else {
+                    log.error("Error while connecting the endpoint. Relieved HTTP response code is : " +
```

Comment:
better if we can log the error details in the reponse, you can access them from `method` object.